--- a/bin.cjs
+++ b/bin.cjs
@@ -16797,6 +16797,13 @@
       Module._resolveFilename = function(request, _parent) {
         const isCoreModule = _module2.builtinModules.includes(request);
         if (!isCoreModule) {
+          if (request.endsWith('.js')) {
+            const tsRequest = request.replace(/\.js$/, '.ts');
+            try {
+              const modifiedArguments2 = [tsRequest, ...[].slice.call(arguments, 1)];
+              return originalResolveFilename.apply(this, modifiedArguments2);
+            } catch (e) {}
+          }
           const found = matchPath(request);
           if (found) {
             const modifiedArguments = [found, ...[].slice.call(arguments, 1)];
